README.md

```diff
@@ -106,7 +106,11 @@ return context.acquire_token_with_authorization_code(
 ``` 
 
 ## Samples and Documentation
-[We provide a full suite of sample applications and documentation on GitHub](https://github.com/AzureADSamples) to help you get started with learning the Azure Identity system. This includes tutorials for native clients such as Windows, Windows Phone, iOS, OSX, Android, and Linux. We also provide full walkthroughs for authentication flows such as OAuth2, OpenID Connect, Graph API, and other awesome features.
+We provide a full suite of [sample applications on GitHub](https://github.com/azure-samples?q=adal) and an [Azure AD developer landing page](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-developers-guide) to help you get started with learning the Azure Identity system. This includes tutorials for native clients such as Windows, Windows Phone, iOS, OSX, Android, and Linux. We also provide full walkthroughs for authentication flows such as OAuth2, OpenID Connect, Graph API, and other awesome features.
```

Comment:
Is there a reason for mentioning the native clients explicitly? If not, I would omit the line or just mention native clients and web applications. 
Secondly,  you can say "walkthroughs for authentication flows such as OAuth2, OpenID Connect and for calling APIs such as the Graph API." This is because the Graph API is not exactly an authentication protocol.